html/index.html

```diff
@@ -6,68 +7,413 @@
-                        <span id="no-char">Select a character slot to see all information about your character.</span>
+                    <div class="characters-list" :class="{'slide-top-left-fade-leave-active': show.characters == false, 'slide-top-left-fade-leave-to': show.characters == false}">
+                        <div class="character-list-header"><p>My Characters</p></div>
+                        <div class="characters">
+                            <template
+                                v-for="index in characterAmount"
+                            >
+                                <div
+                                        v-if="characters[index] == undefined"
+                                        class="character"
+                                        v-ripple
+                                        :class="{'char-selected': selectedCharacter == index}"
+                                        :id="'char-'+ index"
+                                        data-cid=""
+                                        @click="click_character(index, 'empty')"
+                                >
+                                    <span class="slot-name">Empty Slot
+                                        <span class="cid"></span>
+                                    </span>
+                                </div>
+                                <div
+                                        v-if="characters[index] !== undefined"
+                                        class="character"
+                                        v-ripple
+                                        :class="{'char-selected': selectedCharacter == index}"
+                                        :id="'char-'+ index"
+                                        :data-citizenid="characters[index].citizenid"
+                                        @click="click_character(index, 'existing')"
+                                >
+                                    <span class="slot-name">
+                                        {{characters[index].charinfo.firstname}}{{characters[index].charinfo.lastname}}
+                                        <span class="cid">{{characters[index].citizenid}}</span>
+                                    </span>
+                                </div>
+                            </template>
+                        </div>
+                        <div class="character-btns" v-if="selectedCharacter !== -1">
+                            <v-btn large class="character-btn" id="play" v-ripple @click="play_character">
+                                {{characters[selectedCharacter] !== undefined ? "Play" : "Create Character"}}
+                            </v-btn>
+                            <v-btn large class="character-btn" id="delete" v-if="characters[selectedCharacter] !== undefined && allowDelete == true" v-ripple @click="delete_character">
+                                Delete character
+                            </v-btn>
```

Comment:
I suggest making these buttons or just their text bigger, because the text is too small (at least on my screen).

Reply:
I made it larger now.